Code/Framework/AzCore/AzCore/Serialization/SerializeContext.h

```diff
@@ -2496,10 +2496,6 @@ namespace AZ
         template <typename T>
         typename SerializeGenericTypeInfo<T>::ClassInfoType* CreateGenericClassInfo();
 
-        /// Returns GenericClassInfo registered with the current module.
-        template <typename T>
-        AZ::GenericClassInfo* FindGenericClassInfo() const;
-        AZ::GenericClassInfo* FindGenericClassInfo(const AZ::TypeId& genericTypeId) const;
```

Comment:
Looks like I never used these functions when I added them, so it is good with me to remove them.